index.bs

```diff
@@ -602,17 +602,37 @@ After the step
 add the steps
 
 1. Let |attributionReportingEligibility| be "<code>[=eligibility/unset=]</code>".
+1. Let |processResponse| be null.
 1. If |attributionReportingEnabled| is true and the result of [=getting supported registrars=]
      is not [=list/is empty|empty=] and |contextOrigin| [=check if an origin is suitable|is suitable=]:
          1. Set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
+         1. Set |processResponse| to these steps given a [=response=] |response|:
+             1. Run [=process an attribution eligible response=] with |contextOrigin|,
+                 |attributionReportingEligibility|, and |response|.
 
          Issue: Allow "<code>[=eligibility/navigation-source=]</code>" when automatic beacon is specified.
 
-Issue: Set [=request=]'s [=request/Attribution Reporting eligibility=] to |attributionReportingEligibility|
-when <a href="https://wicg.github.io/fenced-frame/#issue-c137ca33">the fetch integration is specified</a>.
+In the step
+
+> Let |request| be a new [=request=] with the following properties...
+
+add the property:
+
+: [=request/Attribution Reporting eligibility=]
+:: |attributionReportingEligibility|
+
+Replace the step
+
+> [=Fetch=] |request|.
+
+with
+
+[=Fetch=] |request| with [=fetch/processResponse=] being |processResponse| if it is not null.
 
 </div>
 
+Issue: Move this section to <a href="https://wicg.github.io/fenced-frame/">Fenced Frame spec</a>.
```

Comment:
I'm surprised to see this given https://github.com/WICG/attribution-reporting-api/pull/848#issuecomment-1598991632. Shouldn't this ARA-specific logic just centralized in this specification?

Reply:
This is from a suggestion from @yoavweiss for the ARA intent to ship.